lis/surfacemodels/land/ac.7.2/ac72_readcrd.F90

```diff
@@ -0,0 +1,264 @@
+    enddo
+
+    ! First day of year of simulation period
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 starting day of sim period:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, AC72_struc(n)%Sim_AnnualStartDay, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 starting day of sim period: not defined")
+    enddo
+
+    ! First month of year of simulation period
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 starting month of sim period:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, AC72_struc(n)%Sim_AnnualStartMonth, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 starting month of sim period: not defined")
+    enddo
+
+    ! First day of year of cropping period
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 starting day of crop period:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, AC72_struc(n)%Crop_AnnualStartDay, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 starting day of crop period: not defined")
+    enddo
+
+
+    ! First month of year of cropping period
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 starting month of crop period:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, AC72_struc(n)%Crop_AnnualStartMonth, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 starting month of crop period: not defined")
+    enddo
+
+    ! PathNameSimul
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 input path:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, &
+            AC72_struc(n)%PathNameSimul, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 input path: not defined")
+    enddo
+
+    ! CO2_Filename
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 CO2_Filename:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, &
+            AC72_struc(n)%CO2_Filename, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 CO2_Filename: not defined")
+    enddo
+ 
+    ! Management_Filename
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 Management_Filename:", rc = rc)
+    do n=1, LIS_rc%nnest
+        if (rc == 0) then
+            call ESMF_ConfigGetAttribute(LIS_config, &
+                AC72_struc(n)%Management_Filename, rc=rc)
+             ! change lis none to AquaCrop (None)
+             if ((AC72_struc(n)%Management_Filename .eq. 'none') .or. &
+                 (AC72_struc(n)%Management_Filename .eq. 'None')) then
+                 AC72_struc(n)%Management_Filename = '(None)'
+             endif 
+        else
+            write(LIS_logunit, *)'[INFO] AC72 Management_Filename: not defined, default management'
+            AC72_struc(n)%Management_Filename = '(None)'
+        endif
+    enddo
+ 
+    ! Irrigation_Filename
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 Irrigation_Filename:", rc = rc)
```

Comment:
This comment probably belongs elsewhere, but I think of it here: is any type of irrigation supported? Perhaps there needs to be a comment in the code on which types are supported (might be there already, will look for it)

Reply:
I'll make that clear in the documentation: ALL kinds of irrigation methods are supported. For now, they are simply given in the .IRR file (same as for the standalone). 
What was added in a few places in the code were lines to manage the reading of those files. AquaCrop tends to keep these files open during the run, but this does not work in LIS. Therefore, I made sure the file was closed each time, and opened at the right line when needed.